### PR TITLE
Create fully functional booklist website

### PR DESCRIPTION
**Pull request by rozhi-98**
In this project, I have finished the booklist web page to be functional.

- Add a navbar and include three different pages.
-  Redirect the user to the page that is clicked from the navbar.
- Make the website responsive.
-  Add copyright and contact info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,63 +4,146 @@
...
+            <p>Do you have any question or you just want to say "hello"?</p>
+            <p>You can reach out to us through:</p>
+          </div>
+          <ul>
+            <li>info@awesomebooks.com</li>
+            <li>Tel: +964-772-218-0404</li>
+            <li>Sulaymaniyah-Kurdistan-Iraq</li>
+          </ul>
+        </div>
+      </section> -->
```
Comment (Santiago220991) [inline index.html]: Kindly remove the commented code. It makes your project looks messy, :+1: :smiley: :smiley:

File: index.html [hunk 2]
```diff
@@ -4,63 +4,146 @@
...
+        <div class="container mt-4">
+          <h2>Contact Information</h2>
+          <p>
+            Do have any question or you just want to say "hello" <br />
+            You can Reach out to us!
+          </p>
+          <ul>
+            <li>Our E-mail: books@awesome.com</li>
+            <li>Our Phone Number: +964-772-218-0404</li>
+            <li>Our Adress: Sulaymaniyah-Kurdistan-Iraq</li>
```
Comment (Santiago220991) [inline index.html:112]: Kindly add a left margin to this text to make your project stick to the template provided.:pray: :pray: 
Your design
image

Template provided
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @rozhi-98, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL] I think is a good idea to group all your module files inside a folder called modules. It will make your project looks neater. :eyes: :eyes: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html:112]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — APPROVED** [review]
Hi @rozhi-98, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rozhi-98** [comment]
@mgmediaweb @Santiago220991 Thank you both for your professional feedback, it is very much appreciated. :pray:

**Pull request merged**